data/data/aws/vpc/variables.tf

```diff
@@ -11,14 +11,10 @@ variable "cluster_id" {
   type = string
 }
 
-variable "private_master_endpoints" {
-  description = "If set to true, private-facing ingress resources are created."
-  default     = true
-}
-
-variable "public_master_endpoints" {
-  description = "If set to true, public-facing ingress resources are created."
-  default     = true
+variable "publish_strategy" {
+  type        = string
+  default     = "External"
```

Comment:
I'd rather not have a default, to ensure we are successfully passing this in from Go.

Reply:
fixed with https://github.com/openshift/installer/compare/586800a7998182fd87bc00700adf24a49df216be...83adc04d6cf70ffc2f45bbe6a5e8d1d06173832d

Reply:
still have a default over [here][1].

[1]: https://github.com/openshift/installer/blob/83adc04d6cf70ffc2f45bbe6a5e8d1d06173832d/data/data/aws/master/variables.tf#L75

Reply:
fixed https://github.com/openshift/installer/compare/09284a49abb5433054f290c35e0aa14a98eef974...2bd7e958c24af56fb51e653786ea3ad37f6d25f4